core/application/commands.go

```diff
@@ -60,6 +65,10 @@ func (a *app) CreateMicroVM(ctx context.Context, mvm *models.MicroVM) (*models.M
 		}
 	}
 
+	if addErr := a.addInstanceData(mvm, logger); addErr != nil {
```

Comment:
I'm not sure there's any benefit in `addErr` here?

You've already returned if `err` wasn't nil earlier, which implies you don't want to retain the value ?

Reply:
That was a mistake but i have changed this to reuse `err`